src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -301,7 +302,14 @@ def _should_report_hydration_error_issue(project: Project) -> bool:
     """
     Checks the project option, controlled by a project owner.
     """
-    return project.get_option("sentry:replay_hydration_error_issues")
+
+    # Killswitch
+    has_feature = features.has(
```

Comment:
why do we need a feature and a killswitch? (on top of the ingest flag created by the issue platform for this issue type)